client/signup/p2-processing-screen/index.jsx

```diff
@@ -1,51 +0,0 @@
-import { useTranslate } from 'i18n-calypso';
-import './style.scss';
-
-function P2SignupProcessingScreen( { signupSiteName = '' } ) {
-	const translate = useTranslate();
-
-	return (
-		<div className="p2-processing-screen">
-			<div className="p2-processing-screen__logo">
-				<img src="/calypso/images/p2/logo-white.png" width="67" height="32" alt="P2 logo" />
```

Comment:
Let's remove these images if they're not used. There are a few of them in this file.

Reply:
Removed in 5ce9df52322